typescript/modules/DescriptionDropzone.ts

```diff
@@ -243,7 +243,7 @@ export class DescriptionDropzone {
         return Object.entries(grouped).map(([category, items]): JQuery => {
             const alert = template.clone();
             alert
-                .children("h4")
+                .children("span")
```

Comment:
I'm testing this with selector `span.alertSubject` instead. It will be too easy for later changes to add more `SPAN` elements to some of these alert templates (e.g. adding FontAwesome elements), and having the broad selector would cause this to replace all of them with the error "title".